### ISSUE TITLE
Two points different from tf, and one may case large cls loss in early stage.

### ISSUE THREAD

**Issue by mashiwei** [zylo117/Yet-Another-EfficientDet-Pytorch#163]
Hi, @zylo117 thank you for your great repos.
I compared your repos with the tf repos, and find two differences.
1.Bias init in cls header, tf use constant initializer, and you use zero_().
https://github.com/google/automl/blob/c91956a724ecda863926fc174b1ce7d97759dc9f/efficientdet/efficientdet_arch.py#L262
https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/104b404430cf0abe8f2732941f082fd46ddfad3b/utils/utils.py#L215
As far as I know, focal loss should use with constant initializer, it can make training more stable.
I modified this function below.
```
            if module.bias is not None:
                if "classifier.header" in name:
                    bias_value = -np.log((1 - 0.01) / 0.01)
                    torch.nn.init.constant_(module.bias, bias_value)
                else:
                    module.bias.data.zero_()
```
The loss is about ten digits in early stage(customer dataset).

2.SeparableConv init in Regressor and Classifier, tf use variance_scaling, and you use kaiming_uniform_.
https://github.com/google/automl/blob/c91956a724ecda863926fc174b1ce7d97759dc9f/efficientdet/efficientdet_arch.py#L227-L228
https://github.com/google/automl/blob/c91956a724ecda863926fc174b1ce7d97759dc9f/efficientdet/efficientdet_arch.py#L284-L285
https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/104b404430cf0abe8f2732941f082fd46ddfad3b/utils/utils.py#L212
I don't know how much this affects. And my modification is below.

I add a init function in torch/nn/init.py
```
def variance_scaling_(tensor, gain=1.):
    # type: (Tensor, float) -> Tensor
    r"""
    VarianceScaling in https://keras.io/zh/initializers/   
    """
    fan_in, fan_out = _calculate_fan_in_and_fan_out(tensor)
    std = math.sqrt(gain / float(fan_in))

    return _no_grad_normal_(tensor, 0., std)
```
and modified this function.
```
            if "conv_list" or "classifier.header" in name:
                nn.init.variance_scaling_(module.weight.data)
            else:
                nn.init.kaiming_uniform_(module.weight.data)
```

**Comment by zylo117** [comment]
great, thanks, can you create a pull request for this?

**Comment by mashiwei** [comment]
> great, thanks, can you create a pull request for this?

a pull request has been created, you may have a check.

**Comment by zylo117** [comment]
PR is merged
